src/kernel/scheduler.zig

```diff
@@ -154,20 +166,20 @@ fn test_fn2() void {}
 
 var test_pid_counter: u7 = 1;
 
-fn task_create(entry_point: EntryPointFn, allocator: *Allocator) Allocator.Error!*Task {
+fn task_create(entry_point: EntryPoint, allocator: *Allocator, kernel: bool, task_vmm: *vmm.VirtualMemoryManager(u8)) Allocator.Error!*Task {
```

Comment:
Have noticed that I forgot the correct function case, so should be tastCreate(). So this can be updated. Same for task_destroy.

Reply:
I've changed it to `createTestTask` and `destroyTestTask` to differentiate them from other functions, what do you think?

Reply:
Sound good